doc/sphinxman/source/dlpnoccsd_t.rst

```diff
@@ -0,0 +1,230 @@
+  2 for full set of diffuse functions, and 1.5 for partial diffuse functions (e.g. if the size limit is 100
+  for cc-pVDZ, expect 30 for cc-pVTZ, 50 for aug-cc-pVDZ, and 70 for jun-cc-pVDZ). Estimates for larger amounts of RAM should be
+  taken with a grain of salt, as the computation may be theoretically possible with DLPNO-CCSD, but may be hindered by the cost of
+  the preceeding Hartree-Fock computation (in time or memory). New approaches to make Hartree-Fock more efficient in |PSIfour| are
+  currently under investigation.
+
+.. _`table:size_limits`:
+
+.. table:: Expected size limits (number of atoms) of DLPNO-CCSD(T) computation
+
+   +------------------+------------+-------------+-------------+
+   | Hardware         | RAM        | Normal      | Tight       |
+   +==================+============+=============+=============+
+   | Home Desktop     | 32 GB      | 90-100      | 80-90       |
+   +------------------+------------+-------------+-------------+
+   | Lab Workstation  | 64 GB      | 120-150     | 100-120     |
+   +------------------+------------+-------------+-------------+
+   | Lab Workstation  | 192 GB     | 300-400     | 200-300     |
+   +------------------+------------+-------------+-------------+
+   | Lab Cluster      | 512 GB     | 1000+       | 500-800     |
+   +------------------+------------+-------------+-------------+
+   | Lab/HPC Cluster  | 1 TB       | 1500+       | 1000+       |
+   +------------------+------------+-------------+-------------+
+   | HPC Cluster      | 3 TB       | 3000+       | 2000+       |
+   +------------------+------------+-------------+-------------+
+
+Key Differences with DLPNO-CCSD(T) in ORCA
+------------------------------------------
+
+We note that while the DLPNO-CCSD(T) algorithm in Psi4 is heavily inspired by the original method
+proposed by Neese and coworkers in ORCA [Riplinger:2013:034106]_ [Riplinger:2013:134101]_ [Riplinger:2016:024109]_ 
+[Guo:2018:011101]_, the DLPNO-CCSD(T) code in |PSIfour| is fundamentally represents a different algorithm than the one
+in ORCA. In other words, the DLPNO-CCSD(T) code in |PSIfour| satisfies the same specifications as the DLPNO-CCSD(T)
+algorithm in ORCA, but represents a different implementation. The best analogy is chicken sandwiches at different fast food chains
+(e.g. Chick-fil-A vs. Popeyes). The food item represents a specification (i.e. fried chicken breast served between buttered, 
+toasted buns), but the implementation is different (different oils, cooking temperatures, breading techniques, seasonings, etc).
+Both algorithms represent linear-scaling CCSD(T) algorithms solved in the the local pair natural orbital basis, with convergence
+to canonical CCSD(T) results as the local tolerances are tightened. However, the manner in which the PNO spaces are truncated as
+well as how the CCSD equations are solved are different. Notable differences in implementation between the two algorithms are
```

Comment:
```suggestion
While the DLPNO-CCSD(T) formulation in |PSIfour| is heavily inspired by the original method
proposed by Neese and coworkers in ORCA [Riplinger:2013:034106]_ [Riplinger:2013:134101]_ [Riplinger:2016:024109]_ 
[Guo:2018:011101]_, in places |PSIfour| employs different algorithms. 
Both represent linear-scaling CCSD(T) algorithms solved in the the local pair natural orbital basis, with convergence
to canonical CCSD(T) results as the local tolerances are tightened. However, the manner in which the PNO spaces are truncated as
well as how the CCSD equations are solved are different. Notable differences in implementation between the two algorithms are
```

Reply:
Ah, I understand the international audience would not understand my Chicken Sandwich reference. To be fair, I was sleep deprived when I wrote some of the docs :)